torch/csrc/api/src/nn/modules/activation.cpp

```diff
@@ -432,7 +432,7 @@ void ThresholdImpl::pretty_print(std::ostream& stream) const {
 
 MultiheadAttentionImpl::MultiheadAttentionImpl(
     const MultiheadAttentionOptions& options_)
-    : Module("torch::nn::MultiheadAttention"), options(options_) {
+    : Cloneable("torch::nn::MultiheadAttention"), options(options_) {
```

Comment:
Why change to cloneable?

Reply:
Because Cloneable is MultiheadAttentionImpl's parent and we have dropped virtual inheritance.